app/views/assessments/show.html.erb

```diff
@@ -13,67 +13,104 @@
 <% @list.delete("submission") %>
 <% @list.delete("reload") %>
 
-<p><i><%= @assessment.description %> </i></p>
-
-<p>
-<b>Due at:</b> <%= due_at_display @aud.due_at %>
-(<b>Last day to handin:</b> <%= end_at_display(@aud.end_at false) %>)
-</p>
-
-<% if @cud.instructor? and @assessment.exam? %>
-<p class="attention">This is an exam. While it is in progress, please check Admin > Edit Course > Exam In Progress.</p>
-<% end %>
-
-<%= render partial: "handin_form", locals: {modal: false, id: "handin_show_assessment"} %>
-
-<table><tr valign=top>
-
-<%# Display general user options %> 
-<td>
-<h2>Options</h2>
-<ul>
-<% @list.sort { |a,b| a[1] <=> b[1] }.each { |key, value| %>
-  <% if value.size > 0  then  %>
-    <li> <%= link_to value, {:action => key}, {:title=>(@list_title[key] || "")} %> </li>
-  <% end %>
-<% } %>
-</ul>
-</td>
+<div class="row">
+  <div class="col-md-12"> 
+    <h2><%= @assessment.name %></h2>
+    <hr>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-4"> 
+    <%# Display any optional instructor admin options %>
+    <% if @cud.instructor? then %>
+      <h3>Admin Options</h3>
+      <ul class="options">
+        <% if @assessment.has_svn then %>
+          <li><%= link_to "SVN Settings", [:adminSVN, @course, @assessment], title: "Manage the SVN repositories" %></li>
+        <% end %>
 
-<%# Display any optional instructor admin options %>
-<% if @cud.instructor? then %>
-  <td width=20></td>
-  <td>
-  <h2>Admin Options (staff)</h2>
-  <ul>
-    <% if @assessment.has_svn then %>
-      <li><%= link_to "SVN Settings", [:adminSVN, @course, @assessment], title: "Manage the SVN repositories" %></li>
+        <% @adminlist.each_pair do |key, value| %>
+            <li> <%= link_to value, {:action => key}, {:title=>(@admin_title[key] || "")} %></li>
+        <% end %>
+      </ul>
     <% end %>
-
-    <% @adminlist.sort { |a,b| a[1] <=> b[1] }.each do |key, value| %>
+    <%# Display the optional CA admin options %>
+    <% if @cud.course_assistant? then %>
+        <h3>Admin Options (CA)</h3>
+        <ul class="options">
+        <% for option in @options do %>
+          <li><%= link_to option['name'], option['url'], :title=>option[:title] %></li>
+        <% end %>
+        </ul>
+    <% end %>
+    <h3>Options</h3>
+    <ul class="options">
+    <% @list.sort { |a,b| a[1] <=> b[1] }.each { |key, value| %>
       <% if value.size > 0  then  %>
-        <li> <%= link_to value, {:action => key}, {:title=>(@admin_title[key] || "")} %></li>
+        <li> <%= link_to value, {:action => key}, {:title=>(@list_title[key] || "")} %> </li>
       <% end %>
+    <% } %>
+    </ul>
+  </div>
+  <div class="col-md-8">
+    <p><i><%= @assessment.description %></i></p>
+    <br>
+    <div class="handin-panel">
+    <p><b>Due at: </b><span class="moment-time" data-format="MMMM Do YYYY, dddd, h:mm:ss a"> <%= due_at_display @aud.due_at %></span></p>
```

Comment:
I would put the weekday first (it feels more natural), and abbreviate the month to three letters: `dddd, MMM Do YYYY, h:mm:ss a` is the format string I'd use. There are a couple of other places this would need to be changed.